docs/source/project_documentation/user_guide/getting_started.md

```diff
@@ -1,14 +1,5 @@
 # Getting Started as a User
 
-## Github Template Repository - Quickstart
-We have a [Github Template Repository](https://github.com/Coffee2Bits/AaC-User-Template-Repository) that's setup to allow users exploring AaC to immediately create an AaC-ready github repository and development environment using [Gitpod](https://gitpod.io/new/#https://github.com/DevOps-MBSE/AaC).
-
-The repository is pre-populated with a simple example model and Github Action workflow. All that you have to do is hit the "Use this Template" button at the top of the template repository. ![Use this Template Button Picture](../../images/github/github_repo_template_button.png)
```

Comment:
Since you removed reference to the `GitHub Template Repository` here, are we also wanting to remove it from being referenced in the [AaC GitOps](https://arch-as-code.org/project_documentation/user_guide/aac_gitops.html#leveraging-aac-in-ci-cd-pipelines) page since it will no longer work with 0.4.0 AaC?